opendatahub-io/odh-dashboard
# Add model bias configuration table

Closes #1269 

## Description


Create the route for the bias metrics configuration page and the list view to visualize it, it's still using the mock data for the configurations.

![Screenshot 2023-05-26 at 9 50 15 AM](https://github.com/opendatahub-io/odh-dashboard/assets/37624318/27cf19e4-d159-412c-b5fd-3f81ceccb393)


## How Has This Been Tested?



You can access it in 2 different pages: project details or global model serving page
1. Go to any of the models, get into the metrics page
2. Since we don't add the button to access the page, we need to change the URL to get into the configuration page. For project details context, it's `projects/:namespace/metrics/model/:inferenceService/configure`. For global page, it's `/modelServing/metrics/:project/:inferenceService/configure`
3. Go to the page and check the table.

## Test Impact


Not yet, can create tests after all the features are done.

## Request review criteria:
